docs/settings.rst

```diff
@@ -16,3 +16,7 @@ Settings
     ``CID_GENERATE``
         Tell the cid middleware to generate a correlation id if it doesn't
         already exist. Default value: ``False``.
+
+    ``CID_SQL_COMMENT_FORMATTER``
```

Comment:
Why make this a function ?
This would be more straightforward to have a simple string here.